crypto/aes/asm/aesv8-armx.pl

```diff
@@ -139,7 +141,8 @@
 	tst	$bits,#0x3f
 	b.ne	.Lenc_key_abort
 
-	adr	$ptr,.Lrcon
```

Comment:
This modification can be isolated by __aarch64__ , because this file is used for arm and aarch64，only aarch64 has instruction `adrp`

Reply:
Sorry, there were some side tracks, so it took me while to get to this. The failure indicates we also use the file when building 32-bit version. So perhaps the right fix is to adjust `adrp` handler in `crypto/perlasm/arm-xlate.pl`. We let perl to emit `adr` instruction when building 32-bit assembly file. I'm going to push the change as follows:
```
diff --git a/crypto/perlasm/arm-xlate.pl b/crypto/perlasm/arm-xlate.pl
index 6281a9b957..13bf68d538 100755
--- a/crypto/perlasm/arm-xlate.pl
+++ b/crypto/perlasm/arm-xlate.pl
@@ -129,7 +129,11 @@ my $adrp = sub {
         # so make sure our asm always uses '#' here.
         #
         $args =~ s/(\w+)#?:lo2:(\w+)/$1#:lo2:$2/;
-        "\tadrp\t$args";
+        if ($flavour =~ /linux32/) {
+            "\tadr\t$args";
+        } else {
+            "\tadrp\t$args";
+        }
     }
 } if (($flavour =~ /ios64/) || ($flavour =~ /linux/));
```